lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -32,13 +38,13 @@ def test_bar3d():
         ax.bar(xs, ys, zs=z, zdir='y', align='edge', color=cs, alpha=0.8)
 
 
-@image_comparison(['bar3d_shaded.png'], remove_text=True)
+@mpl3d_image_comparison(['bar3d_shaded.png'])
 def test_bar3d_shaded():
     x = np.arange(4)
     y = np.arange(5)
     x2d, y2d = np.meshgrid(x, y)
     x2d, y2d = x2d.ravel(), y2d.ravel()
-    z = x2d + y2d
+    z = x2d + y2d + 1  # Avoid triggering bug with zero-depth boxes.
```

Comment:
This seems like a worrying comment.

Reply:
zero-depth boxes don't have stable normal vectors, so sometimes the underside gets shaded, and sometimes the overside gets shaded. this modifies the test slightly so that it won't have zero-sized boxes and we don't have to keep re-baselining it all the time.